src/languages/de-DE/constants.ts

```diff
@@ -0,0 +1,60 @@
+import { Handler } from '#lib/i18n/structures/Handler';
+import { TimeTypes } from '@sapphire/time-utilities';
+
+export class ExtendedHandler extends Handler {
+	public constructor() {
+		super({
+			name: 'de-DE',
+			duration: {
+				[TimeTypes.Year]: {
+					1: 'Jahr',
+					DEFAULT: 'Jahre'
+				},
+				[TimeTypes.Month]: {
+					1: 'Monat',
+					DEFAULT: 'Monate'
+				},
+				[TimeTypes.Week]: {
+					1: 'Woche',
+					DEFAULT: 'Wochen'
+				},
+				[TimeTypes.Day]: {
+					1: 'Tag',
+					DEFAULT: 'Tage'
+				},
+				[TimeTypes.Hour]: {
+					1: 'Stunde',
+					DEFAULT: 'Stunden'
+				},
+				[TimeTypes.Minute]: {
+					1: 'Minute',
+					DEFAULT: 'Minuten'
+				},
+				[TimeTypes.Second]: {
+					1: 'Sekunde',
+					DEFAULT: 'Sekunden'
+				}
+			}
+		});
+	}
+
+	public ordinal(cardinal: number): string {
+		const cent = cardinal % 100;
+		const dec = cardinal % 10;
+
+		if (cent >= 10 && cent <= 20) {
+			return `${cardinal}er`;
+		}
+
+		switch (dec) {
+			case 1:
+				return `${cardinal}er`;
+			case 2:
+				return `${cardinal}er`;
+			case 3:
+				return `${cardinal}er`;
+			default:
+				return `${cardinal}er`;
+		}
```

Comment:
Since all paths just return the number + `er`:
```suggestion
		return `${cardinal}er`;
```